### ISSUE TITLE
Node file logging should rotate logs into archives

### ISSUE THREAD

**Issue by soffokl** [mysteriumnetwork/node#1416]
There is no point to rotate logs if we are keeping it not archived:
```
root@node:~# du -sh /home/node/*
30M	/home/node/mysterium-node.log
34M	/home/node/mysterium-node.log.2019.11.12
215M	/home/node/mysterium-node.log.2019.11.13
224M	/home/node/mysterium-node.log.2019.11.14
260M	/home/node/mysterium-node.log.2019.11.15
201M	/home/node/mysterium-node.log.2019.11.16
260M	/home/node/mysterium-node.log.2019.11.17
232M	/home/node/mysterium-node.log.2019.11.18
```

Old logs should be kept on disk as an archived file, not a plain text.

**Comment by vkuznecovas** [comment]
there should be a sensible limit to the log files, maybe keep the last 2-3 files?

**Comment by Waldz** [comment]
.gz extension probably

**Comment by tadaskay** [comment]
Current "sensible" limit is 7 last days. We can reduce that if we can't afford 1GB of the disk space.

**Comment by tadaskay** [comment]
> There is no point to rotate logs if we are keeping it not archived:
It's not correct there is no point: the point is not to keep logs older than the threshold, i.e. 7 days so we save disk space and make analysis easier. Previous behavior was to retain all of the logs.

**Comment by tadaskay** [comment]
Fixed by #1467
